coop_membership/views/menu.xml

```diff
@@ -258,4 +255,16 @@
         <field name="sequence">10</field>
     </record>
 
+    <record model="ir.ui.menu" id="contacts.menu_contacts">
+        <field name="groups_id" eval="[(3, ref('base.group_user'))]"/>
+    </record>
+
+    <record model="ir.ui.menu" id="website.menu_website_configuration">
+        <field name="groups_id" eval="[(3, ref('base.group_user')), (4, ref('website.group_website_publisher'))]"/>
+    </record>
```

Comment:
This is giving me an error on deployment because website is not installed, nor it shouldn't be installed (not all foodcoops use it)

Why is it added here?